File: Data Structure/TraversalArr.java
// Traversal of an array
/*
    PROCEDURE Traversal (ARRAY x[] OF INTEGER, n: INTEGER)
        FOR i:=0 TO n-1 DO
            PRINT x[i]
        END FOR
    END PROCEDURE    
*/

public class TraversalArr {
    public static void main(String[] args){
        int x[] = {10,20,30,40,50};
        traversal(x,5); 
    }
    static void traversal(int x[], int n){
        for(int i=0; i<n; i++){
            System.out.print(x[i]+" ");
        }
    }
}

File: Data Structure/TraversalArr.cpp
#include <iostream>
using namespace std;
// Traversal of an array
/*
    PROCEDURE Traversal (ARRAY x[] OF INTEGER, n: INTEGER)
        FOR i:=0 TO n-1 DO
            PRINT x[i]
        END FOR
    END PROCEDURE    
*/
void Traversal(int x[], int n) {
    for (int i = 0; i < n; i++) {
        cout << x[i] << " ";
    }
    cout << endl;
}
int main() {
    int arr[] = {1, 2, 3, 4, 5};
    int n = sizeof(arr) / sizeof(arr[0]);
    Traversal(arr, n);
    return 0;
}